### PR TITLE
Fixed fetching the csv file from node, instead of Controller disk

### PR DESCRIPTION
**Pull request by shankarinece**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

Added code to fetch the benchmark file from node, instead of the one on server.

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/benchmark/data/Reader.java [hunk 1]
```diff
@@ -10,13 +10,26 @@
...
 
 public abstract class Reader {
 	
-	public abstract ReadResult read(String file) throws InputException;
+	public abstract ReadResult read(String file, FilePath workspace) throws InputException;
 	
 	@SuppressFBWarnings("DM_DEFAULT_ENCODING")
-	protected InputStreamReader getBufferedReader(String path) throws IOException{
+	protected InputStreamReader getBufferedReader(String path, FilePath workspace) throws IOException{
+//		FilePath f = new FilePath(workspace.)
```
Comment (Lehmann-Fabian) [inline src/main/java/io/jenkins/plugins/benchmark/data/Reader.java]: Please remove the commented code.

**Comment by Lehmann-Fabian** [comment]
Thanks a lot, @shankarinece, for your contribution.
Can you please explain to me why it is necessary?

**Comment by shankarinece** [comment]
> 

Hi @Lehmann-Fabian 
Most of our builds generate benchmark CSV report files on the node's disk. 
The plugin was looking for the file on server's path: https://github.com/jenkinsci/benchmark-evaluator-plugin/blob/master/src/main/java/io/jenkins/plugins/benchmark/data/Reader.java#L20

The change in the plugin is to look for the mentioned file in the node's workspace and if not found, look for it in the server's disk, and so on...

**Comment by Lehmann-Fabian** [comment]
Good point. I'll merge it, but the tests fail. Can you please adjust them accordingly?

**Pull request closed without merging**